config/terraform.spc

```diff
@@ -2,8 +2,8 @@ connection "terraform" {
   plugin = "terraform"
 
   # Paths is a list of locations to search for Terraform configuration files.
-  # Wildcards are supported per https://golang.org/pkg/path/filepath/#Match
+  # Wildcard based searches are supported.
   # Exact file paths can have any name. Wildcard based matches must have an
-  # extension of .tf (case insensitive).
-  # paths = [ "/path/to/dir/*", "/path/to/exact/custom.tf" ]
+  # extension of .tf (case insensitive). Default set to current working directory.
+  paths = [ "./*.tf" ]
```

Comment:
Is there any performance advantage to using `/*.tf` vs. `/*`?

Reply:
@cbruno10 since only `/*` searches all the files (without considering file extension), if a directory has 100+ files in it;
it will search for all files that matches the pattern.
But, if we use `/*.tf`, it will be a bit faster than the previous one, since it is matches all files ends with `.tf`.